### ISSUE TITLE
Problem compiling tag 3.0.0 in x64

### ISSUE THREAD

**Issue by DomenicoSala** [jstedfast/MimeKit#1031]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by DomenicoSala** [comment]
Greetings.
I have a windows application running under ISS [w3wp] 32 bit. For memory reasons we decided to switch to the x64 version [and also the w3wp service now runs at x64] The problem is that when my application tries to access the MimeKit.dll taken from nu-get I get the following error:

File or assembly 'MimeKit, Version=3.0.0.0, Culture=neutral, PublicKeyToken=bede1c8a46c66814' or one of its dependencies could not be loaded. The manifest definition of the specified assembly does not match the assembly reference. (Exception from HRESULT: 0x80131040)

At this point via github I downloaded the sources of the 3.0.0 tag. If I compile ANYCpu everything is fine but if I compile X64 I have a lot of compilation errors. See attached file
![mimeKit_x64](https://github.com/jstedfast/MimeKit/assets/66620384/989392ee-239d-414c-b39c-64f5ea0b8c1b)

How can I get out of this situation and be able to use MimeKit.dll within an X64 environment?

Thanks in advance. Any suggestions will be greatly appreciated

Domenico Giuseppe Sala  [italy]
d.sala@teamsystem.com
